analyze/lib/analyze.js

```diff
@@ -519,6 +519,49 @@ if (0 < ARGUMENTS.length) {
       print(`./cleanup/remove-skeleton-databases.sh <all options you pass to analyze.sh> ${fs.makeAbsolute('skeleton-databases.json')}`);
     }
   };
+  
+  let extractMissingCollections = function (info) {
+    info.missingCollections = [];
+
+    _.each(_.sortBy(info.databases, x => x.name), function (database, name) {
+      let system = database.collections.filter(function (c) {
+        return c.name[0] === '_';
+      }).map(function(c) {
+        return c.name;
+      });
+
+      let missing = [];
+      [ "_apps", "_appbundles", "_aqlfunctions", "_graphs", "_jobs", "_queues" ].forEach(function(name) {
```

Comment:
I bet "_analyzers" is missing in the list

Reply:
@gnusi: that was intentional. It is a script that targets 3.4